buddyboss/buddyboss-platform
# Fixed Video player control not show in white background video

### All Submissions:


Fixes #2562 

### How to test the changes in this Pull Request:

1. Upload any having white video colors
2. Play video and check control bar should be visible

### Proof Screenshots or Video
https://ibb.co/82JvvX2
https://ibb.co/F7NXH42

### Other information:
